src/brushes/pencil_brush.class.js

```diff
@@ -278,6 +358,9 @@
       }
 
       var path = this.createPath(pathData);
+      if (this.simplifyPath) {
```

Comment:
Where is this set?

Reply:
Oops, this is not needed. this.simplifyPath is just the function...so it always passes this check (I guess the user COULD unset it...) I'll remove this check.